app/views/shared/_fallback_links.html.slim

```diff
@@ -0,0 +1,3 @@
+.mt3.mb1
+  - presenter.fallback_links.each do |link|
+    p == link
```

Comment:
Ok so I know I bring this up each time, and it gets kind of confusing, but I believe we'll be better off it we can avoid "blindly" marking strings as HTML-safe (which is what `==` does):

- The issue is that the `recovery_code_link` method translates text and we put a link inside of it, so it becomes HTML text.
- So that means the `recovery_code_fallback.text` key has HTML inside of it, so we need to rename it to end with `html` -- this is a "magic" function of the i18n gem where it will HTML-safe strings for us if the keys are suffixed appropriately
- If we rename the key to `recovery_code_fallback.text_html` then this can be converted to `p =` (I downloaded the branch and tested it locally to confirm)